demo/src/docs/zIndex.js

```diff
@@ -0,0 +1,215 @@
+import React from "react";
+import PrismCode from "react-prism";
+import Alert from "elevate-ui/Alert";
+import Typography from "elevate-ui/Typography";
+import withStyles from "elevate-ui/withStyles";
+import CheckCircle from "elevate-ui-icons/CheckCircle";
+import "../layout/prism-elevate.css";
+
+const zIndex = ({ classes }) => (
+  <div className={classes.wrapper}>
+    <div className={classes.headingContainer}>
+      <Typography type="title" gutterBottom>{`zIndex`}</Typography>
+      <Typography type="body" className={classes.subheading}>
+        This is an example of referencing one of the available values set in our
+        zIndex object:
+      </Typography>
+    </div>
+    <div>
+      <PrismCode component="pre" className="language-javascript">
+        {` 
+    const style = (theme) => ({
+      modal: {
+        zIndex: theme.zIndex.modal,
+      }
+    });
+      `}
+      </PrismCode>
+      <Typography
+        type="body"
+        className={classes.subheading}
+        style={{ fontSize: "18px", marginTop: "24px" }}
+      >
+        Below is what the zIndex object looks like in our theme:
+      </Typography>
+
+      <PrismCode component="pre" className="language-javascript">
+        {` 
+    const zIndex = {
+      buttonBase: 0,
+      buttonChildren: 1,
+      selectDropdown: 2,
+      tableLoading: -1,
+      tableLoadingActive: 2,
+      tableResizer: 10,
+      tooltip: 1000,
+      dropdown: 1500,
+      modal: 2000,
+    };
+      `}
+      </PrismCode>
+      <div />
+    </div>
+
+    <div className={classes.alertContainer}>
+      <Alert color="info">
+        <Typography type="body" style={{ fontSize: "16px" }}>
+          Each property is named after the element it should be set on.
+        </Typography>
+        <Typography type="body" style={{ fontSize: "16px" }}>
+          For example, say you are styling a tooltip, you would simply use the
+          tooltip property within the zIndex object.
+        </Typography>
+      </Alert>
+    </div>
+    <Typography
+      type="body"
+      className={classes.subheading}
+      style={{ fontSize: "18px", marginTop: "16px" }}
+    />
+
+    <Typography
+      type="body"
+      className={classes.subheading}
+      style={{ fontSize: "18px", marginTop: "24px" }}
+    >
+      This allows us to avoid, for example, writing the below just to make sure
```

Comment:
I'm wondering if maybe we should make this message an error alert to make it REAL clear this is something you SHOULD NOT do.